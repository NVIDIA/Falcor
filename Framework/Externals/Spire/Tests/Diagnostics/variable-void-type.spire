// variable with `void` type

void foo()
{
	void a;
}
