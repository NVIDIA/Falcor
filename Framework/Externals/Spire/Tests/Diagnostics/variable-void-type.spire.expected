result code = -1
standard error = {
Tests/Diagnostics/variable-void-type.spire(5): error 30009: invalid type 'void'.
}
standard output = {
}
